fe/fe-core/src/test/java/com/starrocks/sql/analyzer/AnalyzeFunctionTest.java

```diff
@@ -295,4 +295,10 @@ public void testStatistics() throws Exception {
         analyzeFail("select covar_pop(v1,3) from t0");
         analyzeFail("select corr(v1) from t0");
     }
+
+    @Test
+    public void testTableFunction() throws Exception {
+        analyzeSuccess("SELECT * FROM TABLE(generate_series(2, 5, 2));");
+        analyzeSuccess("select * from table(generate_series('2023-12-31 19:00:00', '2024-01-01 00:00:00', interval 3 hour));");
```

Comment:
generate_series can support named arguments, like `generate_series(start = 1, end = 4, step = 3)`, how does generate_series_datetime process step?

Reply:
It processed the same way as ordinary generate_series does. add named arguments case here